README.md

```diff
@@ -43,22 +44,26 @@ See https://github.com/clearcontainers/runtime/wiki/Installation.
 
 See https://github.com/hyperhq/runv.
 
+## Contributors
+
+See the [contributing guide](CONTRIBUTING.md) for details of how to contribute to the project.
+
+# Governance
+
+The Kata Containers project is governed according to the [“four opens"](https://governance.openstack.org/tc/reference/opens.html), which are open source, open design, open development and open community. Technical decisions will be made by technical contributors and a representative Architecture Committee. The community is committed to diversity, openness, encouraging new contributors and leaders to rise up.
```

Comment:
...which are open source, open design, open development, and open community. Technical decisions are made by technical contributors and a representative Architecture Committee. The community is committed to diversity, openness, and encouraging new contributors and leaders to rise up.